pennylane_lightning/core/src/gates/Gates.hpp

```diff
@@ -120,8 +120,8 @@ static constexpr auto getS() -> std::vector<ComplexT<T>> {
 template <template <typename...> class ComplexT, typename T>
 static constexpr auto getT() -> std::vector<ComplexT<T>> {
     return {ONE<ComplexT, T>(), ZERO<ComplexT, T>(), ZERO<ComplexT, T>(),
-            ConstMult(SQRT2<decltype(ONE<ComplexT, T>().x)>() / 2,
-                      ConstSum(ONE<ComplexT, T>(), -IMAG<ComplexT, T>()))};
+            ConstMult(INVSQRT2<ComplexT, T>(),
+                      ConstSum(ONE<ComplexT, T>(), IMAG<ComplexT, T>()))};
```

Comment:
Wait, was this incorrect in the previous definition?

Reply:
`SQRT2<decltype(ONE<ComplexT, T>().x)>() / 2` is correct but a roundabout way to get `INVSQRT2<ComplexT, T>()`. 
`-IMAG<ComplexT, T>()` was a mistake. During the controlled-gate implementation I found many other such [typos] in that module which isn't tested much it seems.